Feature-IOPS-2018-Add-Valuesets-Canonical-URL-to-England-HealthcareService

KazeemHamzat opened the pull request:
Feature-IOPS-2018-Add-Valuesets-Canonical-URL-to-England-HealthcareService

KazeemHamzat committed:
Add Outstanding Assets required NHSE IG V1.0

Add Outstanding Assets required NHSE IG V1.0

KazeemHamzat committed:
Update File Name:ValueSet-England-ERSBookableType

ValueSet-England-ERSBookableType

KazeemHamzat committed:
Add ValueSets URL to England-HealthCareService

Add ValueSets URL to England-HealthCareService

KazeemHamzat committed:
Update England-Organization.xml

Raise binding of Organization.type to "Extensible"

digitalhealthmartin commented on codesystems/CodeSystem-England-ERSBookableType.xml:
File already added in PR https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/375/files

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
File already added in PR https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/375/files

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionTaskStatusReason.xml:
File already added in PR https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/375/files

digitalhealthmartin commented on valuesets/ValueSet-England-ERSBookableType.xml:
File already added in PR https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/375/files

digitalhealthmartin commented on valuesets/ValueSet-England-ERSRequestFlowType.xml:
File already added in PR https://github.com/NHSDigital/NHSEngland-FHIR-ImplementationGuide/pull/375/files

digitalhealthmartin reviewed (changes requested):
Raised points about same files added in different PRs

KazeemHamzat committed:
Remove Assets with minimal or no reference in NHSE IG v1.0

Remove Assets with minimal or no reference in NHSE IG v1.0

KazeemHamzat committed:
Add CodeSystem & ValueSet

1. Added CodeSystem-England-MedicationRequestEndorsement
2. Added ValueSet-England-DMPrescriptionEndorsement

KazeemHamzat committed:
Update England-MedicationRequest.xml

Map ValueSet-England-DMPrescriptionEndorsement to MedicationRequest.extension:dispensingInformation

digitalhealthmartin commented on codesystems/CodeSystem-England-ERSBookableType.xml:
Something looks wrong here as it's gone from ERSPurpose to ERSBookableType (looking at all the changes). Maybe best to delete this file, and create a new file afresh and add the code in it. Hopefully git won't track these sort of changes.

KazeemHamzat replied:
Created a new file, but GitHub sees it the same. 
Have further reviewed with it Declan and considered it okay as long as file content is OK.

digitalhealthmartin commented on codesystems/CodeSystem-England-MedicationRequestEndorsement.xml:
Add to top line: `<?xml version="1.0" encoding="utf-8"?>`

digitalhealthmartin commented on structuredefinitions/England-MedicationRequest.xml:
```diff
@@ -172,6 +172,10 @@
       <isModifier value="false" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="https://fhir.nhs.uk/ValueSet/England-DMPrescriptionEndorsement" />
```
Where does this change come from? Can't find the ticket about it

KazeemHamzat replied:
Good spot. Should have mapped ValueSet-England-DMMedicationDispenseType to Extension.extension:dispenseStatus.
Item 4 of the Feedback email.

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <valueInteger value="0" />
+        </parameter>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 3" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 2" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 5" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 4 (Part I)" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 4 (Part II)" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
```diff
@@ -0,0 +1,80 @@
+            <display value="Schedule 1" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-controlled-drug" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
```diff
@@ -0,0 +1,71 @@
+            <valueInteger value="0" />
+        </parameter>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
```diff
@@ -0,0 +1,71 @@
+            <display value="Contraceptive" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
```diff
@@ -0,0 +1,71 @@
+            <display value="Sexual Health" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
```diff
@@ -0,0 +1,71 @@
+            <display value="Advisory Committee on Borderline Substances" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement" />
```
The system value needs updating

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
```diff
@@ -0,0 +1,71 @@
+            <display value="Selected List Scheme" />
+        </contains>
+        <contains>
+            <system value="https://fhir.nhs.uk/CodeSystem/medicationrequest-endorsement" />
```
The system value needs updating

KazeemHamzat committed:
Changing mapping to ValueSet/England-DMMedicationDispenseType from Profile level to Extension.extension:dispenseStatus

Changing mapping to ValueSet/England-DMMedicationDispenseType from Profile level to Extension.extension:dispenseStatus

digitalhealthmartin commented on valuesets/ValueSet-England-DMMedicationRequestControlledDrug.xml:
Remove the added |1.0.0 version from each system value (wasn't there before).

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionEndorsement.xml:
Remove the added |1.0.0 version from each system value (wasn't there before).

KazeemHamzat committed:
Modification to ValueSets

Remove version number from the url of the valueset options.

digitalhealthmartin commented on valuesets/ValueSet-England-DMPrescriptionTaskStatusReason.xml:
```diff
@@ -0,0 +1,116 @@
+        <offset value="0" />
+        <parameter>
+            <name value="version" />
+            <valueUri value="https://fhir.nhs.uk/CodeSystem/England-MedicationRequestStatusHistory|1.0.0" />
```
Remove |1.0.0

KazeemHamzat committed:
Removing version from CodeSystems Canonical URL

Removing version from CodeSystems Canonical URL
